Add docs for OpenTelemetry Java (#5968)
getsentry/sentry-docs: src/platform-includes/performance/opentelemetry-setup/with-java-agent/without-auto-init/debugging/java.spring-boot.mdx

> To enable debug logging for Sentry, please provide `SENTRY_DEBUG=true` as environment variable or
> add `sentry.debug=true` to your `application.properties`.
> 
> To also show debug output for OpenTelemetry please add `-Dotel.javaagent.debug=true` to the command.

Reviewer: ```suggestion
To enable debug logging for Sentry, set `SENTRY_DEBUG=true` as an environment variable or
add `sentry.debug=true` to your `application.properties`.

To show debug output for OpenTelemetry, add `-Dotel.javaagent.debug=true` to the command.
```
